lib/mayaUsd/fileio/shading/shadingModeExporter.cpp

```diff
@@ -84,6 +84,9 @@ void UsdMayaShadingModeExporter::DoExport(
     const UsdMayaJobExportArgs& exportArgs = writeJobContext.GetArgs();
     const UsdStageRefPtr&       stage = writeJobContext.GetUsdStage();
 
+    if (!exportArgs.exportMaterials)
```

Comment:
The material export is disabled here,

Reply:
After talking with Jerry, I'll remove the material option as it is redundant. I'll re-open the review when done.